web_view_google_map/static/src/js/widgets/gplaces_autocomplete.js

```diff
@@ -329,7 +329,7 @@ odoo.define('web_view_google_map.GplaceAutocompleteFields', function (require) {
                 general: {
                     name: 'name',
                     website: 'website',
-                    phone: ['international_phone_number', 'formatted_phone_number']
+                    // phone: ['international_phone_number', 'formatted_phone_number']
```

Comment:
Hi @osi-scampbell 

Instead of add comment to this line (disable phone), maybe better to override function `_onUpdateWidgetFields` and prevent the widget overwrite existing value if any
the override function will looks like this
```javascript
_onUpdateWidgetFields: function (values) {
    values = values || {};
    if (Object.prototype.hasOwnProperty.call(this.fillfields.general || '', 'phone')) {
        if (this.recordData.phone) {
            delete values['phone'];
        }
    }
    if (Object.prototype.hasOwnProperty.call(this.fillfields.general || '', 'website')) {
        if (this.recordData.website) {
            delete values['website'];
        }
    }
    this._super(values);
}
```